binding/Binding/GRVkInterface.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace SkiaSharp
+{
+	public class GRVkInterface : SKObject, ISKReferenceCounted
+	{
+		[Preserve]
+		internal GRVkInterface (IntPtr h, bool owns)
+			: base (h, owns)
+		{
+		}
+
+		public static GRVkInterface Create (
+			IntPtr vkGetInstanceProcAddr,
+			IntPtr vkGetDeviceProcAddr,
```

Comment:
Instead of having pointers, I think we need actual delegates as well - or maybe replace this version.

Reply:
The way you retrieve those function pointers doesn't depend just on the platform, but also from the loader you use. Not sure what you have in mind to simplify, but I don't see a way that is flexible enough to cover all the needs that doesn't involve passing function pointers provided by the loader.